styles/style.css

```diff
@@ -871,45 +609,29 @@ h1 a {
 
 .nav li:nth-child(4n+0) {
   -webkit-animation: glitch-in 0.3s linear 2s 4;
-  -moz-animation: glitch-in 0.3s linear 2s 4;
-  -ms-animation: glitch-in 0.3s linear 2s 4;
   animation: glitch-in 0.3s linear 2s 4;
   -webkit-animation-fill-mode: both;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [The property -moz-animation is compatible with -webkit-animation and should be included as well.](https://www.codacy.com/app/j-hasseriis/cepheus/file/1888804921/issues/source?bid=3085146&fileBranchId=3088807#l618)